framework_tvb/tvb/tests/framework/core/entities/amodel_manager_test.py

```diff
@@ -46,10 +46,11 @@ class TestsModelManager(BaseTestCase):
     This class contains tests for the tvb.core.entities.modelmanager module.
     """
 
+    @pytest.mark.skipif
     def teardown_method(self):
         init_test_env()
 
-
+    @pytest.mark.skipif
```

Comment:
This one "amodel_manager_test" should be fixed. I see no reason to remove it, or skip it. 
It tests cleaning the env.